CakesROPSpider/src/nvram.c

```diff
@@ -112,7 +112,7 @@ int ApplyPatch(const u8 *patchBuf, u8 *work, u32 sel)
 	if(compat.app.IFile_Open(&file, L"dmc:/ropCustom.txt", FILE_R) == 0)
 	{
 		const u32 maxPathLen = PAYLOAD_FNAME_LEN / 2;
-		const u32 maxFnameLen = maxPathLen - 4;
+		const u32 maxFnameLen = maxPathLen - 5;
```

Comment:
I'd rather set maxPathLen - 1 instead, since the code seems to be written for maxPathLen to be the absolute maximum regardless of the presence of the terminating \0.
Sorry for being so nitpicky, might change that myself once merged.